code/modules/ghostroles/spawnpoint.dm

```diff
@@ -102,7 +102,12 @@ GLOBAL_LIST_EMPTY(ghostrole_spawnpoints)
 			stack_trace("Couldn't find role. Deleting self.")
 			qdel(src)
 		return
-	role.AttemptSpawn(user.client, src)
+	else
+		var/choice = input(user, "Are you certain you wish to activate this pod?", "Control Pod") as null|anything in list("Yes", "No")
```

Comment:
this needs to 
1. use tgui_alert because there's no need for a list
2. return if choice isn't "Yes", not two booleans
3. use a message that asks if you want to spawn as the role's name, because not every role is necessarily a pod

Reply:
I'm updating this, but I'm not entirely sure I've understood what you meant.